src/js/common/schemaValidatedComponent.js

```diff
@@ -0,0 +1,310 @@
+/* globals require */
```

Comment:
I wonder why this passes linting when fluid itself is used as a global on line 310

Reply:
It passes because we have [a rule in our ESLint config to indicate that fluid is always a global](https://github.com/fluid-project/eslint-config-fluid/blob/master/.eslintrc.json#L7).